lib/app/admin/data-reset/test/create-buildData-spec.js

```diff
@@ -0,0 +1,27 @@
+var sinon = require('sinon');
+require('sinon-as-promised');
+
+
+var create = require('./../create');
+
+var stubBuilderCreate = require('./util/stubBuilder-create');
+
+
+describe('Test create', function() {
+  var mediatorRequestStub = sinon.stub();
+  var mediatorMock = {request: mediatorRequestStub};
+
+  beforeEach(function() {
+    mediatorMock.request.reset();
+  });
+
+
+  it('Should build all data', function(done) {
```

Comment:
Whenever testing promises with mocha, don't use `done`. Mocha understands returning a promise. If the promise resolves, test passes, and if it rejects, then test fails.

This applies to a lot of the tests in this PR.